js/components/form/base-form.js

```diff
@@ -136,7 +136,7 @@ export default {
                 }
             },
             errorPlacement: function(error, element) {
-                $(element).closest('.form-group,.field-wrapper').append(error);
+                $(element).closest('.form-group,.field-wrapper').children('div').append(error);
```

Comment:
OK, does it work with both horizontal and vertical form ? If yes, it's fine to me.